openverse_catalog/dags/util/test_pg_cleaner.py

```diff
@@ -9,7 +10,7 @@
 from util.loader import test_sql
 
 
-RESOURCES = os.path.join(os.path.abspath(os.path.dirname(__file__)), "test_resources")
+RESOURCES = Path(__file__).parent.resolve() / "test_resources"
```

Comment:
I love `pathlib`! :star_struck:

Reply:
Me too! Moving the code to pathlib one file at a time :)